### PR TITLE
feat: add per head budget

### LINKED ISSUE #80: feat: add per head budget
- update schema of Trip and add budget field
- add budget field to create form field
- modify mutation for budget

### PR DESCRIPTION
**Pull request by ssarkar551**

### CODE REVIEW

File: src/components/Home/Authenticated/NewTripButton.tsx [hunk 1]
```diff
@@ -244,14 +248,42 @@ export const NewTripButton = () => {
                       " // End date must be after start date!"}
                   </div>
                 </fieldset>
+                <fieldset>
+                  <label htmlFor="perHeadBudget" className="text-lg">
+                    Per Head Budget
```
Comment (AyanavaKarmakar) [inline src/components/Home/Authenticated/NewTripButton.tsx]: Per Head Budget (in ₹) or Per Head Budget (in INR)

maybe something like this

File: src/components/Trip/TripDetailsContainer.tsx [hunk 2]
```diff
@@ -85,6 +85,16 @@ export const TripDetailsContainer: FC<Props> = (props) => {
                     : "loading..."
                 }`}
               </div>
+              <div
+                className={clsx(
+                  "my-2 leading-none",
+                  `${semiBoldFont.className}`,
+                )}
+              >
+                {`Budget Per Head: ${
```
Comment (AyanavaKarmakar) [inline src/components/Trip/TripDetailsContainer.tsx]: Budget: ₹ <amount> per head or Budget: <amount> INR per head

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

File: src/components/Trip/TripDetailsContainer.tsx [hunk 3]
```diff
@@ -85,6 +85,16 @@ export const TripDetailsContainer: FC<Props> = (props) => {
                     : "loading..."
                 }`}
               </div>
+              <div
+                className={clsx(
+                  "my-2 leading-none",
+                  `${semiBoldFont.className}`,
+                )}
+              >
+                {`Budget: INR per head: ${
```
Comment (AyanavaKarmakar) [inline src/components/Trip/TripDetailsContainer.tsx:94]:
```suggestion
                {`Budget: ${
```

File: src/components/Trip/TripDetailsContainer.tsx [hunk 4]
```diff
@@ -85,6 +85,16 @@ export const TripDetailsContainer: FC<Props> = (props) => {
...
+                  isLoading === false ? trip?.budget : "loading..."
+                }`}
```
Comment (AyanavaKarmakar) [inline src/components/Trip/TripDetailsContainer.tsx:96]:
```suggestion
                } per head`}
```

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]
what happened to the changes in the new trip button?

**Comment by ssarkar551** [comment]
> what happened to the changes in the new trip button?

had a merge conflict, file got deleted when merge conflict was resolved

**Pull request closed without merging**